fsharp_core.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If this is named Directory.build.props then it'll get auto-included in any build of sln files under that folder (the netfx.props should probably also be merged into that?)

Reply:
Makes sense, I have added a condition in the file to apply it only on F# projects.

Reply:
Actually, the mono assembly search path has to be referenced as it does not seem to work from the `Directory.Build.props` file. However, I have moved the new bits to it.

Reply:
Ah, sorry for making extra work for you with silly ideas! cool you got it all resolved in the end 👍

Reply:
No worries at all. It would be nice if it worked.